src/clients/chunker.rs

```diff
@@ -15,10 +15,9 @@
 
 */
 
-use std::{collections::HashMap, pin::Pin};
-
 use futures::{Future, Stream, StreamExt, TryStreamExt};
 use ginepro::LoadBalancedChannel;
+use std::{collections::HashMap, pin::Pin};
```

Comment:
Nit: seems like an accidental change? all the `std` imports are typically at the top

Reply:
hmm interesting. I think this happened with formatter or something.. I'll try switching